proto/constraint.proto

```diff
@@ -10,26 +10,6 @@ enum ConstraintType {
   CONSTRAINT_TYPE_FIXED = 1;
   CONSTRAINT_TYPE_MAX = 2;
   CONSTRAINT_TYPE_MIN = 3;
-  CONSTRAINT_TYPE_RANGE = 4;
-  CONSTRAINT_TYPE_WHITELIST = 5;
-  CONSTRAINT_TYPE_MAX_PER_PERIOD = 6;
-  CONSTRAINT_TYPE_MAGIC_CONSTANT = 7;
-}
-
-enum MagicConstant {
-  UNSPECIFIED = 0;
-  VULTISIG_TREASURY = 1;
-}
```

Comment:
Why are we removing the `MagicConstant` type?

We still need this for the resolution of targets and recipient values, yes?